src/gates.jl

```diff
@@ -1,30 +1,94 @@
 module Gates
 
+const gate_alias =
+    [("H_XZ", :H),
```

Comment:
Why is this called H_XZ? Isn't it implied that H swaps X and Z?

Reply:
I just added all the names / aliases that STIM had - this being one of them (like CX == ZCX == CNOT).
There are 3 Hadamard gates in Stim, H_XZ, H_XY, and H_YZ, where H_XZ is an alias for H.